doc/classes/SceneTree.xml

```diff
@@ -61,7 +61,7 @@
 			<param index="2" name="process_in_physics" type="bool" default="false" />
 			<param index="3" name="ignore_time_scale" type="bool" default="false" />
 			<description>
-				Returns a [SceneTreeTimer] which will [signal SceneTreeTimer.timeout] after the given time in seconds elapsed in this [SceneTree].
+				Returns a [SceneTreeTimer] which will emit the [signal SceneTreeTimer.timeout] signal after the given time in seconds elapsed in this [SceneTree].
```

Comment:
```suggestion
				Returns a [SceneTreeTimer] which will emit [signal SceneTreeTimer.timeout] after the given time in seconds elapsed in this [SceneTree].
```
Should be sufficient, otherwise too wordy imo

Reply:
cool